<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MESEditor</class>
 <widget class="QDialog" name="MESEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1125</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редактирование МЭС</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Профиль:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Код:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Наименование:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Описание:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_8">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Длительность:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblKSG">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>КСГ:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="2">
        <widget class="QLineEdit" name="edtName"/>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QLineEdit" name="edtCode"/>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="CRBComboBox" name="cmbKSG"/>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="CRBComboBox" name="cmbGroup">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="2">
        <widget class="QTextEdit" name="textDescription"/>
       </item>
       <item row="5" column="1" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>минимальная</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinMin">
           <property name="maximum">
            <number>1000</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>максимальная</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinMax">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>средняя</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="spinAvg">
           <property name="decimals">
            <number>4</number>
           </property>
           <property name="maximum">
            <double>999.999900000000025</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblEndDate">
           <property name="text">
            <string>Дата окончания</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="CDateEdit" name="edtEndDate">
           <property name="displayFormat">
            <string>dd.MM.yyyy</string>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0" colspan="4">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="topMargin">
          <number>19</number>
         </property>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Модель пациента:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="edtModel"/>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Тариф:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="edtTariff"/>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Норматив визитов:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinKSG">
           <property name="value">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblPeriodicity">
           <property name="text">
            <string>Периодичность</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="edtPeriodicity">
           <property name="maximum">
            <number>999</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="chkActive">
         <property name="text">
          <string>активно</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="chkInternal">
         <property name="text">
          <string>внутренний</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="4">
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tabMKB">
          <attribute name="title">
           <string>МКБ</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="CInDocTableView" name="tblMKB"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tabServices">
          <attribute name="title">
           <string>Услуги</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="CInDocTableView" name="tblServices"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tabVisits">
          <attribute name="title">
           <string>Визиты</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="CInDocTableView" name="tblVisits"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tabEquipment">
          <attribute name="title">
           <string>Оборудование</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="CInDocTableView" name="tblEquipment"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tabMedicament">
          <attribute name="title">
           <string>Медикаменты</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="CInDocTableView" name="tblMedicament"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tabBlood">
          <attribute name="title">
           <string>Препараты крови</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="CInDocTableView" name="tblBlood"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tabNutrient">
          <attribute name="title">
           <string>Питательные средства</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <widget class="CInDocTableView" name="tblNutrient"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tabLimitedBySexAge">
          <attribute name="title">
           <string>Ограничение по полу и возрасту</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="CInDocTableView" name="tblLimitedBySexAge"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="1" column="3">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2">
        <widget class="QCheckBox" name="chkPolyTrauma">
         <property name="text">
          <string>Политравма</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>CInDocTableView</class>
   <extends>QTableView</extends>
   <header>library/InDocTable.h</header>
  </customwidget>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cmbGroup</tabstop>
  <tabstop>cmbKSG</tabstop>
  <tabstop>edtCode</tabstop>
  <tabstop>edtName</tabstop>
  <tabstop>textDescription</tabstop>
  <tabstop>spinMin</tabstop>
  <tabstop>spinMax</tabstop>
  <tabstop>spinAvg</tabstop>
  <tabstop>edtEndDate</tabstop>
  <tabstop>edtModel</tabstop>
  <tabstop>edtTariff</tabstop>
  <tabstop>spinKSG</tabstop>
  <tabstop>edtPeriodicity</tabstop>
  <tabstop>chkActive</tabstop>
  <tabstop>chkInternal</tabstop>
  <tabstop>chkPolyTrauma</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>tblMKB</tabstop>
  <tabstop>tblServices</tabstop>
  <tabstop>tblEquipment</tabstop>
  <tabstop>tblMedicament</tabstop>
  <tabstop>tblBlood</tabstop>
  <tabstop>tblNutrient</tabstop>
  <tabstop>tblLimitedBySexAge</tabstop>
  <tabstop>tblVisits</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MESEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MESEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
